.github/DeliverToNuGet.ps1

```diff
@@ -0,0 +1,156 @@
+Param(
+    [Hashtable] $parameters
+)
+
+function GenerateNuspec
+(
+    [Parameter(Mandatory=$true)]
+    $PackageId,
+    [Parameter(Mandatory=$true)]
+    $Version,
+    [Parameter(Mandatory=$true)]
+    $Authors,
+    [Parameter(Mandatory=$true)]
+    $Owners
+)
+{
+    [xml] $template = '<?xml version="1.0" encoding="utf-8"?>
+    <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+        <metadata>
+            <id></id>
+            <version></version>
+            <title>D365 Business Central — System Modules</title>
```

Comment:
Just shouldn't be hardcoded to System Modules, but for now it is only that module.

Configuring the nuget package needs to be handled per project.